google-cloud-bigtable/lib/google/cloud/bigtable/table.rb

```diff
@@ -145,6 +136,25 @@ def cluster_states
         #
         # @return [Array<Google::Bigtable::ColumnFamily>]
         #
+        # @example
+        #   require "google/cloud/bigtable"
+        #
+        #   bigtable = Google::Cloud::Bigtable.new
+        #
+        #   table = bigtable.table("my-instance", "my-table", perform_lookup: true)
+        #
+        #   table.column_families.each do |cf|
+        #     p cf.name
+        #     p cf.gc_rule
+        #   end
+        #
+        #   # Get column family by name
+        #   cf1 = table.column_families.find_by_name("cf1")
+        #
+        #   # Create column family
+        #   gc_rule = Google::Cloud::Bigtable::GcRule.max_versions(3)
+        #   cf2 = table.column_families.create("cf2", gc_rule)
```

Comment:
Does the method `ColumnFamilyMap#create ` exist?

Reply:
No, this example is one of two that are skipped pending follow-up API changes.

Reply:
`table.column_families.add` also doesn't exist, since `Table#column_families` returns an ordinary array.

Reply:
See #4134 for follow-up that will should changes to this example.